frontend/src/app/commonComponents/RadioGroup.tsx

```diff
@@ -92,7 +97,7 @@ const RadioGroup = ({
           validationStatus === "error" && "usa-form-group--error"
         )}
       >
-        {choices.map((c, i) => (
+        {(choices as []).map((c: any, i: any) => (
```

Comment:
I think this may be happening because `choices` on line 57 doesn't have the right shape when it's an array. Adding `checkedArray: undefined` might fix it? If not, typing `choices` as an `OptionsArray` should do it for sure.

Edit: But see comment below for an alternative.

Reply:
We went with `OptionsArray` bc `checkedArray: undefined` didn't work/created other issues.